cloudformation/synthetics.yaml

```diff
@@ -27,7 +34,7 @@ Resources:
         TimeoutInSeconds: 90
       RuntimeVersion: 'syn-nodejs-puppeteer-3.4'
```

Comment:
Latest version is 'syn-nodejs-puppeteer-3.5'. I think we can also use the latest version